updater/osv/osv.go

```diff
@@ -203,6 +203,9 @@ var ignore = map[string]struct{}{
 	"github actions": {}, // Shouldn't be in containers?
 	"linux":          {}, // Containers have no say in the kernel.
 	"oss-fuzz":       {}, // Seems to only record git revisions.
+	"ubuntu":         {}, // Have a dedicated ubuntu updater.
+	"red hat":        {}, // Have a dedicated Red Hat VEX updater.
+	"suse":           {}, // Have a dedicated SUSE updater.
```

Comment:
Should we consider ones which we *may* have a dedicated updater for in the future? Like `almalinux`, `chainguard`, `rocky linux`, and `wolfi` (ie those which are related to Linux distros which do have their own vuln feeds?

Also, let's include `openSUSE` because I think the current suse support covers both?

Reply:
Probably, even if it was needed from OSV I think we'd be able to remove them from the list without too much hassle. I initially created it as I was testing something else and saw over a million OSV ubuntu rows.